spec/controllers/api/current/patients_controller_spec.rb

```diff
@@ -110,11 +135,33 @@
         expect(PatientPhoneNumber.updated_on_server_since(sync_time).count).to eq 10
         patients_payload.each do |updated_patient|
           updated_phone_number = updated_patient['phone_numbers'].first
-          db_phone_number      = PatientPhoneNumber.find(updated_phone_number['id'])
+          db_phone_number = PatientPhoneNumber.find(updated_phone_number['id'])
           expect(db_phone_number.attributes.with_payload_keys.with_int_timestamps)
             .to eq(updated_phone_number)
         end
       end
+
+      it 'does not change registration user or facility' do
+        current_user = FactoryBot.create(:user)
+        current_facility = FactoryBot.create(:facility)
+        request.env['HTTP_X_USER_ID'] = current_user.id
+        request.env['HTTP_X_FACILITY_ID'] = current_facility.id
+        request.env['HTTP_AUTHORIZATION'] = "Bearer #{current_user.access_token}"
+
+        patients_payload = updated_patients_payload
+
+        previous_registration_user_id = Patient.first.registration_user_id
+        previous_registration_facility_id = Patient.first.registration_facility_id
+
+        post :sync_from_user, params: { patients: patients_payload }, as: :json
+
+        expect(response).to have_http_status(200)
+        patient = Patient.first
+        expect(patient.registration_user.id).to eq previous_registration_user_id
```

Comment:
We can also use change matchers here

https://relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/expect-change

Reply:
Hm, there are multiple assertions here, and they'd all need to revolve around making the `post`? Also, since would be a change we'd need to make across all specs, we could do it in a separate PR?